cacheops/conf.py

```diff
@@ -15,7 +15,8 @@ class Settings(object):
     CACHEOPS_DEFAULTS = {}
     CACHEOPS = {}
     CACHEOPS_LRU = False
-    CACHEOPS_DEGRADE_ON_FAILURE = False
+    CACHEOPS_DEGRADE_ON_FAILURE = True
+    CACHEOPS_DEGRADE_PERSISTENT_PER_REQUEST = False
```

Comment:
CACHEOPS_DEGRADE_TILL_REQUEST_FINISHED sounds ok.
Also there is an another bug here: I shouldn't change default for CACHEOPS_DEGRADE_ON_FAILURE